### PR TITLE
Displaying the current Inquiry

### PR DESCRIPTION
**Pull request by DarynaBorzovets**
ACCEPTANCE CRITERIA

When app starts up, it should query the server for the Inquiry it should display

(Note: can use mock server until the real API is available)

If the app gets back the Inquiry, it should show this Inquiry on the main screen

If the app gets back an error, it should retry at 10 minute intervals. 

This can be done by app scheduling itself to run again in 10 mins and quitting

Or the app can simply not show the main window and silently retry until it succeeds. Either way.

Ticket SR-85 https://incredihire-academy.atlassian.net/jira/software/c/projects/SR/boards/3?selectedIssue=SR-85

### CODE REVIEW

File: Survey/Survey/Models/SurveyViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    func loadInquiries() {
+        mockServer.loadInquiries()
+            .receive(on: DispatchQueue.main)
+            .sink(receiveCompletion: { completion in
+                switch completion {
+                case .finished:
+                    break
+                case .failure(let error):
+                    print("Failed to load inquiries: \(error)")
```
Comment (Eugene-Polonsky) [inline Survey/Survey/Models/SurveyViewModel.swift:17]: where is this printing?

File: Survey/Survey/Models/SurveyViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                DispatchQueue.main.asyncAfter(deadline: .now() + 600) {
```
Comment (Eugene-Polonsky) [inline Survey/Survey/Models/SurveyViewModel.swift:18]: i'm not sure permanent retry every 600ms is the right mechanism here. let's discuss

File: Survey/Survey/Network/db.json
Comment (Eugene-Polonsky) [inline Survey/Survey/Network/db.json:1]: i think this file has to be at the root

File: Survey/Survey/Models/SurveyViewModel.swift (hunk 2 above)
Comment (DarynaBorzovets) [reply Survey/Survey/Models/SurveyViewModel.swift:18]: It is means every 10min as in acceptance criteria

File: Survey/Survey/Network/db.json
Comment (DarynaBorzovets) [reply Survey/Survey/Network/db.json:1]: Nope, it is works perfectly. I have created MockServer. I have not done it though js-server

File: Survey/Survey/Models/SurveyViewModel.swift (hunk 1 above)
Comment (DarynaBorzovets) [reply Survey/Survey/Models/SurveyViewModel.swift:17]: SurveyView, in case failed to load the question

Comment (Eugene-Polonsky) [reply Survey/Survey/Models/SurveyViewModel.swift:17]: There's no console when you're running an application. That only works when you're running a console app, which this isn't. So in this case, you have two options:

1. log to file. There are 3rd party loggers for OSX we can install that will greatly simplify, just google around for a good library.
2. Show something to user. In general this is the preferred route if there's a recovery step they can take. (e.g. connect to the internet). If not, then yeah silent retry is better. But in this case, we don't want to bug the user, so we go straight to silent retry.

Bottom line is I think installing a logger library and using it would be the best approach.

File: Survey/Survey/Models/SurveyViewModel.swift (hunk 2 above)
Comment (Eugene-Polonsky) [reply Survey/Survey/Models/SurveyViewModel.swift:18]: ah okay yeah you're right. I'm so used to these being in ms. :)

File: Survey/Survey/Network/db.json
Comment (Eugene-Polonsky) [reply Survey/Survey/Network/db.json:1]: Sounds good!

File: Survey/Survey.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -80,13 +88,31 @@
...
+				C9253F852C9F6B7E003F075D /* InquiryModel.swift */,
+			);
+			path = Models;
+			sourceTree = "<group>";
+		};
 		C96DB3742C9D0CF900D7001D /* Preview Content */ = {
 			isa = PBXGroup;
 			children = (
 			);
 			path = "Preview Content";
```
Comment (Eugene-Polonsky) [inline Survey/Survey.xcodeproj/project.pbxproj:104]: I'm actually getting a build error building this project saying that this path doesn't exist.

Comment (Eugene-Polonsky) [reply Survey/Survey.xcodeproj/project.pbxproj:104]: Maybe you didn't commit these files?

File: Survey/Survey/SurveyView.swift [hunk 4]
```diff
@@ -68,6 +66,6 @@ struct CustomButtonStyle: ButtonStyle {
 
 struct SurveyView_Previews: PreviewProvider {
     static var previews: some View {
-        SurveyView()
+        SurveyView(question: "How was your day?")
```
Comment (Eugene-Polonsky) [inline Survey/Survey/SurveyView.swift:69]: Note that when I run the app I am not seeing the question show up. There are error messages around sandbox, not sure that's relevant: 

networkd_settings_read_from_file Sandbox is preventing this process from reading networkd settings file at "/Library/Preferences/com.apple.networkd.plist", please add an exception.
networkd_settings_read_from_file Sandbox is preventing this process from reading networkd settings file at "/Library/Preferences/com.apple.networkd.plist", please add an exception.
BOOL _NSPersistentUIDeleteItemAtFileURL(NSURL *const __strong) Failed to stat item: file:///Users/genepol/Library/Containers/Incredihire-LLC.Survey/Data/Library/Saved%20Application%20State/Incredihire-LLC.Survey.savedState/restorecount.plist
